libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3195,6 +3195,12 @@ uint8_t GCS::get_channel_from_port_number(uint8_t port_num)
 MAV_RESULT GCS_MAVLINK::handle_command_request_message(const mavlink_command_int_t &packet)
 {
     const uint32_t mavlink_id = (uint32_t)packet.param1;
+
+    if (mavlink_id == MAVLINK_MSG_ID_MESSAGE_INTERVAL) {
+        const mavlink_command_int_t msg_interval_cmd = { packet.param2, };
```

Comment:
```suggestion
        const mavlink_command_int_t msg_interval_cmd { command : packet.param2, };
```

Sorry, but I don't think we should rely on the magic packing stuff that mavlink does in its messages to do this!

Reply:
Updated.